portfolio/src/main/java/com/google/sps/servlets/CreatePuzzleServlet.java

```diff
@@ -60,27 +65,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // Upload the file to blobstore and get its URL.
     String uploadedFileUrl = uploadToCloudStorage(fileName, fileInputStream);
 
-    //I think that Puzzle will need another field for difficulty
     Puzzle puzzle = new Puzzle();
 
     puzzle.setImageUrl(uploadedFileUrl);
     puzzle.setDifficulty(difficulty);
 
     PuzzleDao dao = new PuzzleDao();
 
-    //Need to double check that puzzle is passed by reference
-    dao.create(puzzle);
+    puzzle = dao.create(puzzle);
 
-    Long puzzleId = puzzle.getPuzzleId();
-    String redirectURL = "/ViewPuzzle?puzzleid=%s" + String.valueOf(puzzleId);
+    // Long puzzleId = puzzle.getPuzzleId();
+    // String redirectURL = "/ViewPuzzle?id=%s" + String.valueOf(puzzleId);
```

Comment:
Don't forget to delete these.

Reply:
I deleted them(I need the changes merged in order to update PuzzleDao)